gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/public/src/components/privacySettings/js/AddAuthorizationDialog.js

```diff
@@ -19,10 +19,10 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
 (function ($, fluid) {
     "use strict";
 
-    fluid.defaults("gpii.oauth2.addAuthorizationDialog", {
+    fluid.defaults("gpii.oauth2.addUserAuthorizedAuthorizationDialog", {
```

Comment:
It looks to me like the `AddAuthorizationDialog.js` file has been replaced by `AddUserAuthorizedAuthorizationDialog.js` and can be removed?